inc/class-command.php

```diff
@@ -70,6 +70,13 @@ class Command {
 	 *
 	 * [--format=<format>]
 	 * : Render output in a particular format.
+	 *
+	 * [--order=<order>]
+	 * : Ascending or Descending order. ASC|DESC.
```

Comment:
Can we use the [YAML doc annotation](https://make.wordpress.org/cli/handbook/commands-cookbook/#annotating-with-phpdoc) to indicate available options?